### PR TITLE
Changes to refactor connector

### PR DESCRIPTION
**Pull request by Ram-srini**
1. Create connector module in sdk and move direct model implementation classes to connector.
2. Create blockchains module in connector and move fabric, ethereum implementation classes to blockchains.
3. Fix imports

Signed-off-by: Ramakrishna Srinivasamurthy <ramakrishna.srinivasamurthy@intel.com>

### CODE REVIEW

File: docs/dev-environments/fabric/README.md [hunk 1]
```diff
@@ -20,19 +20,19 @@ curl -o minifab -sL https://tinyurl.com/twrt8zv && chmod +x minifab
 The Fabric connection profile is vars/mychannel_connection.json which is needed for application to connect to Fabric network
 
 
-### Extract Avalon [chaincode](https://github.com/hyperledger/avalon/tree/master/sdk/avalon_sdk/fabric/chaincode) and deploy them on the Fabirc network
+### Extract Avalon [chaincode](https://github.com/hyperledger/avalon/tree/master/sdk/avalon_sdk/connector/blockchains/fabric/chaincode) and deploy them on the Fabirc network
```
Comment (danintel) [inline docs/dev-environments/fabric/README.md:23]: Change Fabirc to Fabric

File: docs/dev-environments/fabric/README.md [hunk 2]
```diff
@@ -20,19 +20,19 @@ curl -o minifab -sL https://tinyurl.com/twrt8zv && chmod +x minifab
...
 
 ```
 ./getandinstall.sh
 ```
 
 
 ### Run your tests
-Once you have all the chaincode installed and connection profile available, you can use [Avalon python connector libary](https://github.com/hyperledger/avalon/tree/master/sdk/avalon_sdk/fabric) to make Avalon API 1.1 calls.
+Once you have all the chaincode installed and connection profile available, you can use [Avalon python connector libary](https://github.com/hyperledger/avalon/tree/master/sdk/avalon_sdk/connector/blockchains/fabric) to make Avalon API 1.1 calls.
```
Comment (danintel) [inline docs/dev-environments/fabric/README.md:31]: Change libary to library

File: docs/dev-environments/fabric/README.md (hunk 1 above)
Comment (Ram-srini) [reply docs/dev-environments/fabric/README.md]: Done

File: docs/dev-environments/fabric/README.md (hunk 2 above)
Comment (Ram-srini) [reply docs/dev-environments/fabric/README.md]: Done

**Pull request merged**